### DISCUSSION TITLE
How to insert an definition block?

### DISCUSSION THREAD

**Discussion by Photonico** [marp-team/marp#469] [Q&A]
The definition block is a very important element in STEM slides. 
How to insert a definition block with rounded square background?

**Comment by yhatt** [comment]
Marp allows you to apply your preferred styles to specific Markdown elements using the `<style>` tag. Example:

```
> Blockquote as definition block

<style>
blockquote {
  border: 0;
  border-radius: 1em;
  background: #ccc;
  color: currentcolor;
  padding: 1em;
  width: fit-content;
}
</style>
```

Marp team is not a design consultancy, so we cannot detailed offer based on the current information. :)
